### ISSUE TITLE
not working with python 3.9

### ISSUE THREAD

**Issue by tosaric** [PatrikHlobil/Pandas-Bokeh#123]
pandas bokeh is still not compatible with python 3.9 inside conda environment, right?
I am trying to install pandas bokeh via conda install -c patrikhlobil pandas-bokeh and I am getting following error:

```
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                       

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - pandas-bokeh -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

**Comment by PatrikHlobil** [comment]
Hi @tosaric,

I had some problems providing the conda packages for pandas bokeh for python 3.9. Can you install it via pip?

Best Patrik

**Comment by tosaric** [comment]
> Hi @tosaric,
> 
> I had some problems providing the conda packages for pandas bokeh for python 3.9. Can you install it via pip?
> 
> Best Patrik

Yes, I can install it via pip. Thanks for your response. Don't mind me for using this comment to express my gratitude for such an amazing plotting library. Great work.

**Comment by joewalp** [comment]
Hi @PatrikHlobil,

What problems did you encounter upon attempting to provide conda packages for pandas bokeh for python 3.9+? Please let me know if you'd like some assistance diagnosing the problem. (I arrived at this issue report after failing to install the library for Python 3.9.7 using mamba.)

**Comment by PatrikHlobil** [comment]
Hi @joewalp,

I do not remember exactly, but there were some conflicting dependencies for the dependencies bokeh and pandas. I guess these should be resolved now. I have to find some time to try and build the packages again.

Best Patrik
